packages/flutter_test/lib/src/controller.dart

```diff
@@ -879,6 +923,27 @@ class LiveWidgetController extends WidgetController {
     await binding.endOfFrame;
   }
 
+  @override
```

Comment:
this implementation needs a test

Reply:
It is tested in `'Test pumpAndSettle on LiveWidgetController'` from `packages/flutter_test/test/live_widget_controller_test.dart`